tests/end_to_end/test_docker_monitoring/test_docker_monitoring.py

```diff
@@ -0,0 +1,65 @@
+import json
+import os
+import re
+import pytest
+from tempfile import gettempdir
+
+from wazuh_testing import end_to_end as e2e
+from wazuh_testing import event_monitor as evm
+from wazuh_testing.tools import configuration as config
+
+# Test cases data
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+test_cases_path = os.path.join(test_data_path, 'test_cases')
+test_cases_file_path = os.path.join(test_cases_path, 'cases_test_docker_monitoring.yaml')
+alerts_json = os.path.join(gettempdir(), 'alerts.json')
+
+# Playbooks
+configuration_playbooks = ['configuration.yaml']
+events_playbooks = ['generate_events.yaml']
+
+# Configuration
+configurations, configuration_metadata, cases_ids = config.get_test_cases_data(test_cases_file_path)
+
+
+@pytest.mark.parametrize('metadata', configuration_metadata, ids=cases_ids)
+@pytest.mark.filterwarnings('ignore::urllib3.exceptions.InsecureRequestWarning')
+def test_docker_monitoring(configure_environment, metadata, get_dashboard_credentials, generate_events,
+                           clean_environment):
+    rule_description = metadata['rule.description']
+    rule_id = metadata['rule.id']
+    docker_action = metadata['extra']['data.docker.Action']
+    expected_api_alert = f".+\"Action\": \"({docker_action})\".+\"description\": \"({rule_description})\".+\"id\": " \
+                         f"\"({rule_id})\""
+    expected_log_alert = f".+\"description\":\"({rule_description})\".+\"id\":\"({rule_id})\"" \
+                         f".+\"Action\":\"({docker_action})\""
```

Comment:
We need to obtain the timestamp, take a look at: https://github.com/wazuh/wazuh-qa/blob/81d6c7259522275c88767e0fb531661e9a3036a4/tests/end_to_end/test_audit/test_audit.py#L35-L39

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/2948/commits/36b2041ccde824b50f63d79895dc0ba2dfbf12c4